src/webob/acceptparse.py

```diff
@@ -339,7 +362,7 @@ def _parse_media_type_params(cls, media_type_params_segment):
                 value = cls._process_quoted_string_token(token=value)
                 media_type_params[index] = (name, value)
 
-        return media_type_params
+        return tuple(media_type_params)
```

Comment:
The return value has been changed to a tuple, but the docstring is still saying that it returns a list?